### ISSUE TITLE
Publish OSLC4NET.Core on NuGet

### ISSUE THREAD

**Issue by berezovskyi** [OSLC/oslc4net#16]
We should not require people to clone OSLC4NET and build the DLL before starting to use it.

Actually, we should publish the following packages on NuGet:

- [x] OSLC4NET.Core
- [x] OSLC4NET.Core.Providers.RdfXml
- [x] OSLC4NET.Core.Providers.Json
- [ ] OSLC4NET.Core.Domains.Config
- [x] OSLC4NET.Client

**Comment by heku** [comment]
That would be great!

**Comment by berezovskyi** [comment]
The client is now usable with https://www.nuget.org/packages/OSLC4Net.Client/0.3.0-alpha and the following code:

```csharp
private const string OSLC_SERVER_URI = "https://oslc.itm.kth.se/ccm";

static void Main(string[] args)
{
	try
	{
		var helper = new JazzRootServicesHelper(OSLC_SERVER_URI, OSLCConstants.OSLC_CM_V2);
		var catUri = helper.GetCatalogUrl();
		Console.WriteLine($"The OSLC server has an OSLC Service Provider Catalog as the following URI:\n    {catUri}");
	} catch (RootServicesException e) {
		Console.WriteLine($"Failed to fetch the OSLC RootServices document from:\n    {OSLC_SERVER_URI}/rootservices");
	}
	Console.ReadLine();
}
```

Fixed by #72
